src/button/button-group.scss

```diff
@@ -6,45 +6,53 @@
   display: inline-flex;
   align-items: center;
 
-  > .iui-button {
+  $childSelector: '&, input, button'; // target nested inputs and buttons
+
+  > * {
     &:not(.iui-borderless) {
-      border-radius: 0;
-    }
+      #{$childSelector} {
+        border-radius: 0;
+      }
+
+      &:first-child {
```

Comment:
I remember we changed to first-of-type/last-of-type because split-menu has a tippy as last-child.

But we can't use first-of-type/last-of-type anymore because of the mixed nature of these combo groups. So we might need to move the menu into a separate div.

Reply:
What if users have Tooltips on buttons inside button group? Then it won't work too and we don't control that.

Reply:
I think the users will just need to move the tooltip outside the div 🤷‍♀️

Reply:
But it will be kind of a breaking change then and not really obvious one.

Reply:
I added wrapper divs to each child. So now if there is a need to put tooltips on a disabled button, it will work like this:
```jsx
<div.button-group>
  <div> // This div will be added to all children regardless
    <div> // Tooltip component will wrap this element, but the DOM will be appended as a sibling
      <button disabled />
    </div>
  </div>
   ...
</div>